connectors/http/rest/element-templates/hybrid/http-json-connector-hybrid.json

```diff
@@ -391,16 +391,51 @@
   }, {
     "id" : "connectionTimeoutInSeconds",
     "label" : "Connection timeout in seconds",
-    "description" : "Sets the timeout in seconds to establish a connection or 0 for an infinite timeout",
-    "optional" : true,
+    "description" : "Defines the connection timeout in seconds, or 0 for an infinite timeout",
+    "optional" : false,
     "value" : "20",
+    "constraints" : {
+      "notEmpty" : true
+    },
     "feel" : "optional",
     "group" : "timeout",
     "binding" : {
       "name" : "connectionTimeoutInSeconds",
       "type" : "zeebe:input"
     },
     "type" : "String"
+  }, {
+    "id" : "readTimeoutInSeconds",
+    "label" : "Read timeout in seconds",
+    "description" : "Timeout in seconds to read data from an established connection or 0 for an infinite timeout",
```

Comment:
we use HTTP verbs below for the POST/PUT, would be nice to do the same here (GET)

Reply:
Great thanks :)